Source/Terminals.Plugins.Web/WebOptionsConverter.cs

```diff
@@ -1,18 +1,31 @@
 ﻿using Terminals.Common.Connections;
 using Terminals.Common.Converters;
+using Terminals.Data;
 
 namespace Terminals.Plugins.Web
 {
-    internal class WebOptionsConverter : IOptionsConverter
+    internal class WebOptionsConverter : OptionsConverterTemplate<WebOptions>, IOptionsConverter
     {
-        public void FromConfigFavorite(OptionsConversionContext context)
+        protected override void FromConfigFavorite(FavoriteConfigurationElement source, WebOptions options)
         {
-            UrlConverter.UpdateFavoriteUrl(context.Favorite, context.ConfigFavorite.Url);

```

Comment:
Why are this property conversions removed? The property is still there, so it needs to be converted.

Reply:
I'll look into it :)